builtin_interfaces/Time start_time
uint16 msg_seq
uint16 original_num
